### DISCUSSION TITLE
Add an E2 core for adv dupe2

### DISCUSSION THREAD

**Discussion by JohnWeak1** [wiremod/advdupe2#432] [Suggestions]
it would be nice if there would be an e2 core for adv dupe2

where you could have functions like :
runOnAdvDupe(n)
r = LastDupeEntities()
r = LastDupeConstraints()
e = LastDupeOwner()
and more if that's possible.

**Comment by thegrb93** [comment]
Will label wontfix until someone decides they want to do the work.

**Comment by vurvdev** [comment]
Easy fix now with the events system, best time for anyone to do it
